doc/tech/release_notes/baldy.md

```diff
@@ -0,0 +1,121 @@
+[<< Back](../)
+# Baldy Release Notes
+
+
+## Baldy Release Contents
+* [Overview](#overview)
+* [RM (v3.0)](#rm)
+* [RA-1 (v2.0)](#ra1)
+* [RI-1 (v1.0)](#ri1)
+* [RC-1 (v1.0-pre-alpha)](#rc1)
+* [RA-2 (v1.0)](#ra2)
+
+<a name="overview"></a>
+## Overview
+
+| Reference # 	| Feature 	| Notes 	|
+|-------------	|-----------------	|-------	|
+| `baldy.tech.1` 	| VNF Revolution policy and strategy 	|  	|
+| `baldy.tech.2` 	| Future Roadmap |  |
+| `baldy.tech.3` 	| TBD |  |
+
+*Baldy Release should include at least features up to and including `baldy.tech.2`.
+
+<a name="rm"></a>
+## Reference Model (v3.0)
+
+>WSL Note: Features below should be implemented in order.
+
+| Reference # 	| Feature 	| Notes 	|
+|-------------	|-----------------	|-------	|
+| `baldy.rm.1` 	| General Cleanup 	| All Chapters 	|
+| `baldy.rm.2` 	| Limiting infrastructure profiles to Basic and Network Intensive (Parking Compute intensive) | Ch02, Ch04, and Ch05 |
+| `baldy.rm.3` 	| Finalising Compliance, Verification and Conformance Strategy | Ch08 	|
+| `baldy.rm.4*` 	| Full Container support 	| Ch04 	|
+| `baldy.rm.5` 	| Complete Security Chapter | Ch07: 100% alignment with ONAP 	|
```

Comment:
Is ONAP alignment for Security really necessary? RM CH07 security deals with NFVI while ONAP security deals with security that VNFs needs to comply with and incorporate.

Reply:
agree but as the outcome of our cross collaboration, it will make sense to make sure our security chapter doesn't conflict with their VNF security guidelines (agree they have different focus). it is pretty much high in priority as it is not a lot of work and will be good collaboration result that can be celebrated in our next cross community discussion.

Reply:
I agree